### PR TITLE
Update 5.0-contributor-list.md

### PR DESCRIPTION
**Pull request by rbhanda**

### CODE REVIEW

File: release-notes/5.0/5.0-contributor-list.md [hunk 1]
```diff
@@ -1,1176 +1,1176 @@
...
+</br>[sdk](#sdk)
+</br>[templating](#templating)
+</br>[wcf](#wcf)
+</br>[windowsdesktop](#windowsdesktop)
+</br>[winforms](#winforms)
+</br>[wpf](#wpf)
+</br>
+</br>## aspnetcore
+</br>
+</br>[= johluo@microsoft.com (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=johluo@microsoft.com)
```
Comment (MichaelSimons) [inline release-notes/5.0/5.0-contributor-list.md]:
```suggestion
</br>[johluo@microsoft.com (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=johluo@microsoft.com)
```

File: release-notes/5.0/5.0-contributor-list.md [hunk 2]
```diff
@@ -1,1176 +1,1176 @@
...
+</br>[David Fowler (29)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=davidfowl@gmail.com)
+</br>[David Melendez (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dlmelendez@users.noreply.github.com)
+</br>[davidnx 44079650+davidnx@users.noreply.github.com (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=44079650+davidnx@users.noreply.github.com)
+</br>[Dawid Szmidka (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dawid.szmidka@gmail.com)
+</br>[Dennis Daume (1)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dennis@getmimo.com)
+</br>[DotNet Bot (21)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dn-bot@microsoft.com)
+</br>[dotnet-maestro-bot dotnet-maestro-bot@microsoft.com (5)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=dotnet-maestro-bot@microsoft.com)
+</br>[dotnet-maestro[bot] 42748379+dotnet-maestro[bot]@users.noreply.github.com (755)](https://github.com/dotnet/aspnetcore/commits/release/5.0?author=42748379+dotnet-maestro[bot]@users.noreply.github.com)
```
Comment (MichaelSimons) [inline release-notes/5.0/5.0-contributor-list.md:71]: Thoughts on removing our well known bots like `dotnet-maestro`?

File: release-notes/5.0/5.0-contributor-list.md [hunk 3]
```diff
@@ -1,1176 +1,1176 @@
...
+</br>[installer](#installer)
+</br>[runtime](#runtime)
+</br>[sdk](#sdk)
+</br>[templating](#templating)
+</br>[wcf](#wcf)
+</br>[windowsdesktop](#windowsdesktop)
+</br>[winforms](#winforms)
+</br>[wpf](#wpf)
+</br>
+</br>## aspnetcore
```
Comment (MichaelSimons) [inline release-notes/5.0/5.0-contributor-list.md]: The `</br>` breaks the MD rendering and links.  This comment applies elsewhere.

```suggestion
## aspnetcore
```

File: release-notes/5.0/5.0-contributor-list.md (hunk 2 above)
Comment (rbhanda) [reply release-notes/5.0/5.0-contributor-list.md]: bots from this list are removed

**Comment by MichaelSimons** [comment]
Looks much better.  Thanks.

**Pull request merged**